### PR TITLE
[Disk Manager] improve nemesis logging

### PR DESCRIPTION
**Pull request by BarkovBG**

### CODE REVIEW

File: cloud/tasks/test/nemesis/main.go [hunk 1]
```diff
@@ -138,6 +162,13 @@ func main() {
 		"maximum time (in seconds) between two consecutive restarts",
 	)
 
+	rootCmd.Flags().StringVar(
+		&restartTimingsFileName,
+		"restart-timings-file-name",
+		"",
+		"file name to store restart timings",
```
Comment (SvartMetal) [inline cloud/tasks/test/nemesis/main.go:169]: file where to store restart timings

File: cloud/disk_manager/test/recipe/disk_manager_launcher.py [hunk 2]
```diff
@@ -315,6 +316,8 @@ def __init__(self,
                 str(min_restart_period_sec),
                 "--max-restart-period-sec",
                 str(max_restart_period_sec),
+                "--restart-timings-file-name",
```
Comment (SvartMetal) [inline cloud/disk_manager/test/recipe/disk_manager_launcher.py:319]: --restart-timings-file

File: cloud/tasks/test/nemesis/main.go [hunk 3]
```diff
@@ -60,6 +61,21 @@ func waitIteration(
...
+				restartTimingsFileName,
+				os.O_APPEND|os.O_WRONLY|os.O_CREATE,
+				0644,
+			)
+			if err != nil {
+				return err
+			}
+			defer f.Close()
+
+			_, err = f.WriteString(time.Now().String())
```
Comment (SvartMetal) [inline cloud/tasks/test/nemesis/main.go:76]: Log error, please

File: cloud/tasks/test/nemesis/main.go [hunk 4]
```diff
@@ -138,6 +165,13 @@ func main() {
 		"maximum time (in seconds) between two consecutive restarts",
 	)
 
+	rootCmd.Flags().StringVar(
+		&restartTimingsFileName,
```
Comment (SvartMetal) [inline cloud/tasks/test/nemesis/main.go:169]: restartTimingsFile

File: cloud/tasks/test/nemesis/main.go [hunk 5]
```diff
@@ -112,10 +133,16 @@ func main() {
 	var cmdString string
 	var minRestartPeriodSec uint32
 	var maxRestartPeriodSec uint32
+	var restartTimingsFileName string
```
Comment (SvartMetal) [inline cloud/tasks/test/nemesis/main.go:136]: restartTimingsFile

File: cloud/tasks/test/nemesis/main.go [hunk 6]
```diff
@@ -49,6 +49,7 @@ func waitIteration(
 	errors chan error,
 	minRestartPeriodSec uint32,
 	maxRestartPeriodSec uint32,
+	restartTimingsFileName string,
```
Comment (SvartMetal) [inline cloud/tasks/test/nemesis/main.go:52]: restartTimingsFile

**Review by SvartMetal — APPROVED** [review]

**Review by EvgeniyKozev — APPROVED** [review]

**Pull request merged**